crypto/ecdsa/signer.go

```diff
@@ -75,3 +75,36 @@ func (x SignerRFC6979) Sign(data []byte) ([]byte, error) {
 func (x SignerRFC6979) Public() neofscrypto.PublicKey {
 	return (*PublicKeyRFC6979)(&x.PublicKey)
 }
+
+// StaticSigner emulates real sign and contains already precalculated hash.
+// Provides neofscrypto.Signer interface.
+type StaticSigner struct {
+	sig    []byte
+	pubKey ecdsa.PublicKey
+}
+
+// NewStaticSigner creates new StaticSigner.
+func NewStaticSigner(sig []byte, pubKey ecdsa.PublicKey) *StaticSigner {
```

Comment:
i don't think it's a good idea to place static signer in `crypto/ecdsa` package. It's better to place it in `crypto` package and use dynamic scheme:
```suggestion
func NewStaticSigner(scheme neofscrypto.Scheme, pubKey neofscrypto.PublicKey, sig []byte) *StaticSigner {
```

we also may optimize usage. Currently signing routines will serialize data to be signed but it aint gonna be used since `StaticSigner.Sign` ignores parameter. In total we do idle serialization. To avoid this we can extend
https://github.com/nspcc-dev/neofs-sdk-go/blob/0bc98456e36176769d01b41eeab69b6143e6b945/crypto/signature.go#L54-L60
with lazy serialization
```
func (x *Signature) CalculateLazy(signer Signer, dataSrc func() []byte) error {
  if static, ok := signer.(*StaticSigner); ok {
    // just set fields as they are
  }

  x.Calculate(signer, dataSrc())
}
```

Thoughts?

Reply:
> NewStaticSigner

:+1:

> CalculateLazy

:-1: It's an optimization we better leave for the future.

Reply:
Moved StaticSigner to `crypto`